Del redundant-static-def in executorch/schema/extended_header.cpp +2

r-barnes opened the pull request:
Summary:
LLVM has a warning `-Wdeprecated-redundant-constexpr-static-def` which raises the warning:

> warning: out-of-line definition of constexpr static data member is redundant in C++17 and is deprecated

Since we are now on C++20, we can remove the out-of-line definition of constexpr static data members. This diff does so.

 - If you approve of this diff, please use the "Accept & Ship" button :-)

Differential Revision: D81359733

JacobSzwejbka reviewed (changes requested):
Review automatically exported from Phabricator review in Meta.

r-barnes committed:
Del redundant-static-def in executorch/schema/extended_header.cpp +2 (#13875)

Summary:

LLVM has a warning `-Wdeprecated-redundant-constexpr-static-def` which raises the warning:

> warning: out-of-line definition of constexpr static data member is redundant in C++17 and is deprecated

Since we are now on C++20, we can remove the out-of-line definition of constexpr static data members. This diff does so.

 - If you approve of this diff, please use the "Accept & Ship" button :-)

Differential Revision: D81359733

JacobSzwejbka reviewed (approved):
Misunderstood the comment on c++20
